lua/terrortown/menus/gamemode/equipment/base_equipment.lua

```diff
@@ -179,9 +179,11 @@ function CLGAMEMODESUBMENU:Populate(parent)
         database = DatabaseElement(accessName, itemName, "damageScaling"),
     })
 
+    local equipmentClass = WEPS.GetClass(equipment)
```

Comment:
Isnt this only intended for weapons?

I think this should not be used for items too

Reply:
we always use this for both in our code base and it works as well. But yes, the name is misleading